### PR TITLE
bug 1685558, use carryforward flags to deal with skipped test runs

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

**Comment by Pike** [comment]
I have no idea why this fails to get the reports processed, opened https://community.codecov.io/t/trying-to-introduce-flags-triggers-error-on-processing-reports/2450

File: .codecov.yml [hunk 1]
```diff
@@ -1 +1,18 @@
 comment: off
+
+flags:
+  backend:
+    paths:
+      - /pontoon/**.py
```
Comment (thomasrockhu) [inline .codecov.yml:6]:
```suggestion
      - pontoon/**.py
```
I think this might work

Comment (Pike) [reply .codecov.yml]: Thanks, that seems to have done the trick. I pushed each path individually, and the backend tests already are marked as carry-forward until the ones from the current test run made it in.

**Comment by Pike** [comment]
This is ready for review, per Tom's comment on discourse.

**Review by mathjazz — APPROVED** [review]
\o/

**Pull request merged**